{"name": "test_buurten_failure", "status": "passed", "description": "Failure API request for the buurten", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/buurten/0363098000001 to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1646844234382, "stop": 1646844234517}], "start": 1646844234382, "stop": 1646844234518, "uuid": "b3228668-1b2b-444b-b90a-3b1ee428e3e9", "historyId": "6d2d2afe8521e49849a7d1201a789880", "testCaseId": "76f46efd1f9a250d2bb8382d9d249708", "fullName": "e2etest.test_datasetgebiedenbuurten#test_buurten_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenbuurten"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "59427-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenbuurten"}]}